src/modules/governance/pages/daoMembersPage/daoMembersPage.tsx

```diff
@@ -31,11 +32,17 @@ export const DaoMembersPage: React.FC<IDaoMembersPageProps> = async (props) => {
     const daoUrlParams = { id: daoId };
     const dao = await queryClient.fetchQuery(daoOptions({ urlParams: daoUrlParams }));
 
-    const { address: bodyPluginAddress } = daoUtils.getDaoPlugins(dao, {
+    const plugins = daoUtils.getDaoPlugins(dao, {
         type: PluginType.BODY,
         includeSubPlugins: true,
-    })![0];
+    });
 
+    if (!plugins?.length) {
+        const daoUrl = daoUtils.getDaoUrl(dao, 'dashboard')!;
+        redirect(daoUrl);
```

Comment:
```suggestion
        redirect(daoUrl);
        return;
```

Add return to avoid errors if redirect is async.